styleguide/source/assets/scss/03-organisms/_resource-tabs.scss

```diff
@@ -191,6 +191,10 @@ li {
     @include gutter($margin-top-full...);
     @include gutter($margin-bottom-half...);
     @include gutter($padding-left-quarter...);
+
+    &.ama_resource-header--highlight {
+      background-color: #dad0e1;
```

Comment:
Can you update the color with the variable `$bodyTextHoverPurple` ?

Reply:
Oh good catch, thanks!